README.md

```diff
@@ -3,8 +3,13 @@
 [![Nuget](https://img.shields.io/nuget/v/KS.fiks.maskinporten.client.svg)](https://www.nuget.org/packages/KS.Fiks.Maskinporten.Client)
 [![GitHub issues](https://img.shields.io/github/issues-raw/ks-no/kryptering-dotnet.svg)](//github.com/ks-no/fiks-maskinporten-client-dotnet/issues)
 
-.net core library for maskinporten authorization through ID-porten
 
+## About this library
+This library is a .net core library for maskinporten authorization through ID-porten.
```

Comment:
Maskinporten og ID-porten er to ulike produkter.
Usikker på om man bør legge på authentication begrepet her.

This is a .Net core library for Maskinporten authentication and authorization.

Reply:
fixed